src/components/organisms/FeatureHeaderSection/FeatureHeaderSection.tsx

```diff
@@ -126,15 +129,23 @@ const FeatureHeaderSection: FC<FeatureHeaderSectionProps> = ({
         machineTranslation={mt_enabled}
         setMachineTranslation={toggleInputChange}
       />
-      <Button
-        appearance={AppearanceTypes.Text}
-        className={classes.addButton}
-        iconPositioning={IconPositioningTypes.Left}
-        icon={Add}
-        children={t('button.add_new_vendor')}
-        onClick={addVendor}
-        hidden={activeTab === FeatureTabs.Xliff || !isSplittingAllowed}
-      />
+      <div className={classes.splitSection}>
+        <Button
+          appearance={AppearanceTypes.Text}
+          className={classes.addButton}
+          iconPositioning={IconPositioningTypes.Left}
+          icon={Add}
+          children={t('button.add_new_vendor')}
+          onClick={addVendor}
+          loading={isLoading}
+          hidden={activeTab === FeatureTabs.Xliff || !isSplittingAllowed}
```

Comment:
since both hidden props have the same value, maybe you can extract it a variable and pass two both of them